src/appliedsciencestudio/covid19_clj_viz/india.clj

```diff
@@ -137,3 +142,221 @@
+                                              :titleColor "#85C5A6"}}}}
+                       {:mark {:stroke "#d32f2f"
+                               :type "tick"}
+                        :encoding {:y {:aggregate "average"
+                                       :field "beds"
+                                       :type "quantitative"
+                                       :axis {:title "No. of hospital beds"
+                                              :titleColor "#d32f2f"}}}}]
+               :resolve {:scale {:y "independent"}}}))
+
+;-------------------------------------------------------------------------------------------------
+
+;;cloropleth: India map visualizing the hospital beds distribution per 1k population of each state
+
+(def india-geojson-with-beds-data
+  (update (json/read-value (java.io.File. "resources/public/public/data/india-all-states.geo.json")
+                           (json/object-mapper {:decode-key-fn true}))
+          :features
+          (fn [features]
+            (mapv (fn [feature]
+                    (let [state (:NAME_1 (:properties feature))
+                          beds (get state-hospital-beds state)]
+                      (assoc feature
+                        :State         state
+                        :beds          beds
+                        :beds-per-1k   (double (Math/ceil (/ beds
+                                                             (/ (get state-population state)
+                                                                1000)))))))
+                  features))))
+
+(comment
+  (json/write-value (java.io.File. "resources/public/public/data/india-all-states-created-with-beds.geo.json")
+                    india-geojson-with-beds-data)
+
+  ;; for inspection without flooding my REPL, we ignore the many many coordinates:
+  (map #(dissoc % :geometry) (:features india-geojson-with-beds-data)))
+
+(oz/view!
+  (merge-with merge oz-config india-dimensions
+              {:title {:text "Current India hospital beds count per 1k population"}
+               :data {:name "india"
+                      :values india-geojson-with-beds-data
+                      :format {:property "features"}},
+               :mark {:type "geoshape" :stroke "white" :strokeWidth 1}
+               :encoding {:color {:field "beds-per-1k",
+                                  :type "quantitative"
+                                  :scale {:field "cases-per-100k",
+                                          :scale {:type "log"}
+                                          :type "quantitative"}}
+                          :tooltip [{:field "State" :type "nominal"}
+                                    {:field "beds-per-1k" :type "quantitative"}]}
+               :selection {:highlight {:on "mouseover" :type "single"}}}))
+
+;------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Let's use the standard "four semicolons, space, 75 equals signs" here.

    ;;;; ===========================================================================